config_test.go

```diff
@@ -0,0 +1,243 @@
+package config
+
+import (
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"path"
+	"testing"
+	"time"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/wawandco/fako"
+)
+
+const (
+	testTomlName        = "test.toml"
+	testTomlDefaultName = "test-default.toml"
+	testTomlFileDir     = "/"
+)
+
+func newFs(t *testing.T, file string) func() {
+	fs = afero.NewMemMapFs()
+	b, err := ioutil.ReadFile(file)
+	require.NoError(t, err)
+	filePath := path.Join(testTomlFileDir, configFileName)
+	require.NoError(t, afero.WriteFile(fs, filePath, b, 0644))
+	f := path.Join(os.TempDir(), filePath+".lock")
```

Comment:
please call this `lockFilePath` or similar to make it clear what it is. At first I thought `f` was relating to the config file itself.